pkg/util/k8s/pod.go

```diff
@@ -32,3 +45,57 @@ func GetPodContainerNames(pod *v1.Pod) []string {
 	}
 	return names
 }
+
+var (
+	netdefutilsSetNetworkStatus = netdefutils.SetNetworkStatus
+	netdefutilsGetNetworkStatus = netdefutils.GetNetworkStatus
+)
+
+func UpdatePodAnnotation(kubeClient clientset.Interface, ctx context.Context, netStatus []netdefv1.NetworkStatus, podName, podNamespace string, isPrimary bool) error {
```

Comment:
Probably better to name it `UpdatePodNetworkStatusAnnotation`

Reply:
done